src/App.php

```diff
@@ -58,7 +58,13 @@ private function getDispatcher()
     {
         $dispatcher = $this->container->get('dispatcher');
         if (!is_callable($dispatcher)) {
-            throw new \RuntimeException('Dispatcher must be a callable');
+            throw new RuntimeException('Dispatcher must be a callable');
+        }
```

Comment:
as the check now check Dispatcher instance of Penny\Dispatcher, this check is not needed, yes?